supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,6 +10,34 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
 
+[[conscious-language]]
+== Conscious language
+
+The Conscious Language Group is supporting the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects that Red Hat is involved with.
```

Comment:
@ndeevy s/is supporting/supports

s/that Red Hat is involved with/with which Red Hat is involved